repos/system_upgrade/common/actors/checkfips/actor.py

```diff
@@ -33,3 +37,11 @@ def process(self):
                     reporting.Groups([reporting.Groups.SECURITY]),
                     reporting.Groups([reporting.Groups.INHIBITOR])
                 ])
+        else:
+            if fips_info.is_enabled:
+                fips_required_initramfs_files = [
+                    '/usr/lib64/ossl-modules/fips.so',
```

Comment:
Is this always going to be `/usr/lib64` for all supported architectures, or do we support systems where this might be `/usr/lib`? Should we use `rpm --eval '%{_libdir}'` to determine this path?

Reply:
We support `x86_64`, `aarch64`,`ppc64le`, `s390x` (all are 64bit). Just to be sure I've checked that `fips.so` provided by the `openssl-libs` is present in the `/usr/lib64` on all of the supported archs.